docs/en/observability/view-monitor-status.asciidoc

```diff
@@ -61,7 +61,7 @@ image::images/monitors-list.png[Monitors list]
 
 [discrete]
 [[observability-integrations]]
-== Integrate with Logs, Metrics, and APM
+== Integrate with Logs, Infrastructure, and APM
```

Comment:
Does it make sense to move to something more like "Integrate with Observability" or "Integrate with Other Observability Apps" etc?